rin_serin/week04/mission02/src/App.jsx

```diff
@@ -0,0 +1,90 @@
+import './App.css'
+import HomePage from './pages/HomePage';
+import {createBrowserRouter, RouterProvider} from "react-router-dom";
+import RootLayout from './layout/root-layout';
+import LoginPage from './pages/LoginPage';
+import SignupPage from './pages/SignupPage';
+import Search from './pages/SearchPage';
+import CategoriPage from './pages/CategoriPage';
+import MovieList from './components/MovieList';
+import MovieIdPage from './pages/MovieIdPage';
+import NowplayingPage from './pages/categori/NowplayingPage';
+import PopularPage from './pages/categori/PopularPage';
+import TopratedPage from './pages/categori/TopratedPage';
+import UpcomingPage from './pages/categori/UpcomingPage';
+
+
+const router  = createBrowserRouter([
+  {
+    path:'/',
+    element:<RootLayout/>,
+    children:[
+      {
+        index: true,
+        element: <HomePage/>
+      },
+      {
+        path:'movies',
+        element: <p>영홥니다</p>
+      },
+      {
+        path:'movies/:movieId',
+        element: <MovieIdPage/>
+      },
+      {
+        path:'movies/now-playing/movies/:movieId',
+        element: <MovieIdPage/>
+      },
```

Comment:
```suggestion
      {
        path:'movies/:movieId',
        element: <MovieIdPage/>
      },
```

해당 부분은 movies/:movieId 경로만 사용하셔도 돼요! 
now-playing 페이지에서 이동할 때는 절대 경로로 이동하면 movies/:movieId로 이동해집니다

참고자료 
👉 https://velog.io/@gud_wns/router-%EC%83%81%EB%8C%80%EA%B2%BD%EB%A1%9C-%EC%A0%88%EB%8C%80%EA%B2%BD%EB%A1%9C%EC%99%80-Link%EC%9D%98-relative-%EC%86%8D%EC%84%B1